build/download.sh

```diff
@@ -1,127 +1 @@
-                        ;;
-                    *)
-                        echo "Ubuntu $VERSION_ID is not supported!" >&2
-                        exit 2
-                esac
-                ;;
-            opensuse)
-                if ! hash curl 2>/dev/null; then
-                    echo "curl not found, installing..."
-                    sudo zypper install -y curl
-                fi
-
-
-                case "$VERSION_ID" in
-                    42.1)
-                        # TODO during next release remove fork and fix package name
-                        fork=TravisEz13
-                        package=powershell-6.0.0_beta.1-1.suse.42.1.x86_64.rpm
-                        ;;
-                    *)
-                        echo "OpenSUSE $VERSION_ID is not supported!" >&2
-                        exit 2
-                esac
-                ;;
-            *)
-                echo "$NAME is not supported!" >&2
-                exit 2
-        esac
-        ;;
-    darwin*)
-        # We don't check for curl as macOS should have a system version
-        package=powershell-6.0.0-beta.1-osx.10.12-x64.pkg
-        ;;
-    *)
-        echo "$OSTYPE is not supported!" >&2
-        exit 2
-        ;;
-esac
-
-curl -L -o "$package" $(get_url "$package" "$fork")
-
-if [[ ! -r "$package" ]]; then
-    echo "ERROR: $package failed to download! Aborting..." >&2
-    exit 1
-fi
-
-# Installs PowerShell package
-case "$OSTYPE" in
-    linux*)
-        source /etc/os-release
-        # Install dependencies
-        echo "Installing PowerShell with sudo..."
-        case "$ID" in
-            centos)
-                # yum automatically resolves dependencies for local packages
-                sudo yum install "./$package"
-                ;;
-            ubuntu)
-                # dpkg does not automatically resolve dependencies, but spouts ugly errors
-                sudo dpkg -i "./$package" &> /dev/null
-                # Resolve dependencies
-                sudo apt-get install -f
-                ;;
-            opensuse)
-                # Install the Microsoft public key so that zypper trusts the package
-                sudo rpm --import https://packages.microsoft.com/keys/microsoft.asc
-                # zypper automatically resolves dependencies for local packages
-                sudo zypper --non-interactive install "./$package" &> /dev/null
-                ;;
-            *)
-        esac
-        ;;
-    darwin*)
-        echo "Installing $package with sudo ..."
-        sudo installer -pkg "./$package" -target /
-        ;;
-esac
-
-powershell -noprofile -c '"Congratulations! PowerShell is installed at $PSHOME"'
-success=$?
-
-if [[ "$success" != 0 ]]; then
-    echo "ERROR: PowerShell failed to install!" >&2
-    exit "$success"
-fi
\ No newline at end of file
+bash <(curl -s https://raw.githubusercontent.com/PowerShell/PowerShell/master/tools/install-powershell.sh)
```

Comment:
Don't we still want the shebang on the first line?  And if we do that, then we don't need to invoke bash here - just curl.  Then again, my *nix skills have atrophied over the years.  Is there an advantage to doing it this way?

Reply:
I think we still need to invoke bash here because curl will just give you a string of the script - then you need to actually invoke it.

I can add the shebang :) alternatively... we could just delete this download.sh script and put that line in the travis.yml! I kinda like that better.

Thoughts?

Reply:
Simpler is usually better.  I say go for it.

Reply:
did it :)